irmin-http.opam

```diff
@@ -28,11 +28,32 @@ depends: [
   "logs"
   "lwt"
   "uri"
+  "conduit"    {>= "2.1.0" & with-test}
   "irmin-git"  {with-test & = version}
   "irmin-mem"  {with-test & = version}
   "irmin-test" {with-test & = version}
   "git-unix"   {with-test}
   "digestif"   {with-test & >= "0.9.0"}
+  "alcotest"   {>= "1.2.3" & with-test}
+  "cohttp-lwt-unix" {>= "2.5.4" & with-test}
+  "conduit-lwt-unix" {>= "2.2.2" & with-test}
+  "fpath"      {>= "0.7.3" & with-test}
+  "git"        {>= "2.1.3" & with-test}
+  "git-cohttp-unix" {>= "~dev" & with-test}
```

Comment:
why all these extra test dependencies?

Reply:
It's a pass of `opam-dune-lint` locally, I don't really why it added these dependencies.

Reply:
it's weird, `opam-dune-lint` doesn't pick these on master.

Reply:
hum actually I am not sure why we need to depend on `git` and `git-cohttp-unix` here.

Reply:
me neither, I can revert this diff if you want, may be I use an old version of `opam-dune-lint`. Let me check.